Swashbuckle-Swagger-Operation-Alphabetical-Order.md

```diff
@@ -0,0 +1,73 @@
+# Arranging operations alphabetically in Swashbuckle Swagger
+
+[Swashbuckle](https://github.com/domaindrivendev/Swashbuckle) seamlessly adds a [Swagger](http://swagger.io/) to WebAPI projects. However, the operations on the page do not appear in alphabetical order. However, Swashbuckle allows custom filters and with a little code, this is easy enough to implement. The code below creates a [DocumentFilter](https://github.com/domaindrivendev/Swashbuckle#documentfilter) which is then applied to the final document as outlined [here](https://github.com/domaindrivendev/Swashbuckle#modifying-generated-operations).
```

Comment:
IS there a way to reword this? Something like "Although the operations on the will not appear in alphabetical order, Swashbuckle allows custom filters and with a little coding, the order can be shown in alphabetical order."

If possible I would just try to avoid using However, in two subsequent sentences.